inception-recommendation/src/main/java/de/tudarmstadt/ukp/inception/recommendation/render/RecommendationSpanRenderer.java

```diff
@@ -224,4 +232,20 @@ public void render(CAS aCas, VDocument vdoc, AnnotatorState aState,
             }
         }
     }
+
+    /**
+     * Describe this annotation by its label or feature
+     * 
+     * @param aSuggestion
+     *            suggestion for an annotation
+     * @return the suggestion's label or feature
+     */
+    private String getAnnotationDescriptor(AnnotationSuggestion aSuggestion)
+    {
+        String label = aSuggestion.getLabel();
+        if (label == null) {
+            label = aSuggestion.getFeature();
+        }
+        return label;
```

Comment:
Could there be a problem if somebody uses the same name for the label as the feature has. E.g. if a recommender is defined over a feature named `value` and for some reason `value` is also a valid label?